topological_utils/scripts/LoadPointSet.py

```diff
@@ -7,57 +7,96 @@
 
 from topological_navigation.topological_node import *
 from strands_navigation_msgs.msg import TopologicalNode
+from strands_navigation_msgs.msg import TopologicalMap
+from strands_navigation_msgs.msg import TopMap
+from strands_navigation_msgs.msg import TopNode
```

Comment:
These lines still need to be removed. Also, can you change the script name to load_point_set.py instead of the CamelCase?